### DISCUSSION TITLE
Multiplayer button isn't there

### DISCUSSION THREAD

**Discussion by Absal0n78** [Papela/Nitrox-Cracked-Mod#21] [Q&A] [answered]
Hello!
First of all, thank you for the mod. :)
When I click on "play" multiplayer with the console running behind, but the game's interface is the classic one (solo).
How can I fix this  please?

**Comment by Papela** [comment, marked as answer]
What version of the game are you using? I recommend using: **mar-2023 71288**
On very new or very old versions the mod does not work properly.

**Reply by Absal0n78** [reply]
I'm using v70023.
I will check for the 71288, thanks!

**Reply by Absal0n78** [reply]
I download the v71288 and it works very well now, thank you!
